Tasks/app-store-release/app-store-release.ts

```diff
@@ -165,6 +165,8 @@ async function run() {
             tl.debug('Skipped fastlane installation.');
         }
 
+        let fastlaneArguments: string = tl.getInput('fastlaneArguments');
```

Comment:
nit: `const` over `let`

Reply:
There are other spots in this file too, will update all of them next time.